statsd/uds.go

```diff
@@ -31,6 +31,18 @@ func newUDSWriter(addr string, writeTimeout time.Duration, transport string) (*u
 	return writer, nil
 }
 
+// GetTransportName returns the transport used by the writer
+func (w *udsWriter) GetTransportName() string {
+	if w.transport == "unixgram" {
+		return writerNameUDS
+	} else if w.transport == "unix" {
+		return writerNameUDSStream
+	} else {
+		// For backward compatibility.
+		return writerNameUDS
+	}
```

Comment:
I think this comment may be misleading, the only way this branch is taken if we'd have `unixpacket` transport, for which there is nothing to be backward compatible with.
```suggestion
	if w.transport == "unix" {
		return writerNameUDSStream
	} else {
		return writerNameUDS
	}
```

Reply:
applied